python/flightsql/gizmosql/main.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2025 Columnar Technologies Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# /// script
+# requires-python = ">=3.9"
+# dependencies = ["adbc-driver-flightsql>=1.8.0", "pyarrow>=20.0.0"]
```

Comment:
This should be `adbc-driver-manager` instead of `adbc-driver-flightsql`.
Please check the dremio example.

Reply:
I will try to get the example working with the slim image...